starlite/response/file.py

```diff
@@ -97,7 +97,7 @@ def __init__(
         stat_result: Optional["stat_result_type"] = None,
         status_code: int = HTTP_200_OK,
     ) -> None:
-        """This class allows streaming a file as response body.
+        """Initialize `FileResponse`
```

Comment:
```suggestion
        """Initialize `FileResponse`.
```

Reply:
This is weird, that should have been caught by `pydocstyle`. Wonder why it ignores that.